bin/adhoc-scripts/parseUnifiedCampaigns.py

```diff
@@ -251,11 +260,37 @@ def upload(mgr, campRec):
         return
     campaign = campRec.get('CampaignName', '')
     if campaign:
-        mgr.postCampaignConfig(campaign, campRec)
+        res = mgr.postCampaignConfig(campaign, campRec)
+        print(res)
     else:
         print("ERROR: found a campaign record without a CampaignName value: %s" % campRec)
 
 
+def insertTestCampaigns(mgr):
+    """
+    Insert a few campaigns configuration used for testing purposes
+    NOTE: the campaign name might change according to the tests you're running.
+    :param mgr: ReqMgrAux object
+    """
+    if not mgr:
+        return
+
+    msg = "\nShould we also insert a few testing campaigns (y/n between quotes): "
+    resp = input(msg)
```

Comment:
I always against inputs from stdin since they lead to problems during scripting. Plus here the input can't be ignored by script input parameters. My suggestion either provide a flag as input parameter to this function along with adding this flag to argument list, or (Better) get rid of input at all and rely on arguments passed to the script. For instance, 
```
parseUnififiedCampaigns.py --add-test-campaigns
```
this is MUCH clear, more robust to user inputs and does not require input from the user interactively